bot/exts/info/pypi.py

```diff
@@ -52,22 +48,15 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     info = response_json["info"]
 
                     embed.title = f"{info['name']} v{info['version']}"
-                    embed.url = info['package_url']
-                    embed.colour = next(PYPI_COLOURS)
 
-                    for field in FIELDS:
-                        field_data = info[field]
+                    embed.url = info["package_url"]
+                    embed.colour = next(PYPI_COLOURS)
 
-                        # Field could be completely empty, in some cases can be a string with whitespaces, or None.
-                        if field_data and not field_data.isspace():
-                            if '\n' in field_data and field == "license":
-                                field_data = field_data.split('\n')[0]
+                    summary = escape_markdown(info["summary"])
 
-                            embed.add_field(
-                                name=field.replace("_", " ").title(),
-                                value=escape_markdown(field_data),
-                                inline=False,
-                            )
+                    # Summary could be completely empty, or just whitespace.
+                    if summary and not summary.isspace():
+                        embed.description = summary
```

Comment:
Could we possibily have a "No Description Provided" for packages with no descriptions?

Reply:
Yes.